### ISSUE TITLE
Regression in 2.0.0-alpha.11

### ISSUE THREAD

**Issue by ukasiu** [software-mansion/react-native-screens#230]
Hi,

it seems that on 2.0.0-alpha.11 after putting app to background we're getting:

```
11-22 08:50:34.971 22825 22825 E AndroidRuntime: java.lang.IllegalStateException: Can not perform this action after onSaveInstanceState
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at androidx.fragment.app.FragmentManagerImpl.checkStateLoss(FragmentManagerImpl.java:1536)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at androidx.fragment.app.FragmentManagerImpl.enqueueAction(FragmentManagerImpl.java:1558)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at androidx.fragment.app.FragmentManagerImpl.popBackStack(FragmentManagerImpl.java:258)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at com.swmansion.rnscreens.ScreenStack.onDetachedFromWindow(ScreenStack.java:65)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at android.view.View.dispatchDetachedFromWindow(View.java:17611)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at android.view.ViewGroup.dispatchDetachedFromWindow(ViewGroup.java:3729)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at android.view.ViewGroup.dispatchDetachedFromWindow(ViewGroup.java:3715)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at android.view.ViewGroup.dispatchDetachedFromWindow(ViewGroup.java:3715)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at android.view.ViewGroup.dispatchDetachedFromWindow(ViewGroup.java:3715)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at android.view.ViewGroup.dispatchDetachedFromWindow(ViewGroup.java:3715)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at android.view.ViewGroup.dispatchDetachedFromWindow(ViewGroup.java:3715)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at android.view.ViewGroup.dispatchDetachedFromWindow(ViewGroup.java:3715)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at android.view.ViewRootImpl.dispatchDetachedFromWindow(ViewRootImpl.java:3736)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at android.view.ViewRootImpl.doDie(ViewRootImpl.java:6835)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at android.view.ViewRootImpl.die(ViewRootImpl.java:6805)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at android.view.WindowManagerGlobal.removeViewLocked(WindowManagerGlobal.java:493)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at android.view.WindowManagerGlobal.removeView(WindowManagerGlobal.java:424)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at android.view.WindowManagerImpl.removeViewImmediate(WindowManagerImpl.java:206)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:5087)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at android.app.ActivityThread.-wrap6(Unknown Source:0)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1955)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:108)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:166)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:7425)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:245)
11-22 08:50:34.971 22825 22825 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:921)
```

The bug is not present in 2.0.0-alpha.9 and seems to be introduced in 47658d4d7d12b2927761a2af35eb2975a77b712d

Using:
react-native 0.61.2
Huawei Mate 10 Lite

**Comment by tobiaslins** [comment]
Experiencing the same with `2.0.0-alpha-12`

**Comment by micheleb** [comment]
same with `2.0.0-alpha-13` and `2.0.0-alpha-18`, and confirming `2.0.0-alpha.9` not being affected

**Comment by WoLewicki** [comment]
@janicduplessis @tobiaslins can I close this issue since it is fixed by #24 ?

**Comment by janicduplessis** [comment]
Sure!
